pkg/logging/fields.go

```diff
@@ -1,11 +1,18 @@
 package logging
 
 import (
+	"context"
+
 	administrationResources "github.com/stackrox/rox/pkg/administration/events/resources"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
```

Comment:
I think we tend to only have two import sections, right?

Reply:
fixed